squidlib-util/src/main/java/squidpony/squidmath/RNG.java

```diff
@@ -349,6 +349,7 @@ public String toString() {
      * @param <T> can be any non-primitive type.
      * @return a shuffled copy of elements
      */
+    @GwtIncompatible
```

Comment:
I need to set up GWT for local testing, but this should be compatible with GWT 2.7.0, released over a year ago it seems. Either that or GWT's documentation is claiming to implement the methods used by this function, but not implementing them fully?